frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -1,11 +1,13 @@
 import * as React from 'react';
 import { ListGroup } from 'react-bootstrap';
-import { IBuilding, IEvaluation, IParcel } from 'actions/parcelsActions';
+import { IParcel } from 'actions/parcelsActions';
 import { Label } from 'components/common/Label';
 import './InfoSlideOut.scss';
 import { formatMoney } from 'utils/numberFormatUtils';
-import { compareDate, OuterRow } from './InfoContent';
+import { OuterRow } from './InfoContent';
 import { ThreeColumnItem } from './ThreeColumnItem';
+import { EvaluationKeys } from 'constants/evaluationKeys';
+import { getCurrentYearEvaluation } from 'features/projects/common/projectConverter';
```

Comment:
I need to extract this into a util (as well as some other financial logic). Fine in this review.